docs/sync.md

```diff
@@ -1,50 +1,35 @@
-# Syncing course content to the live website
+# Syncing course content
```

Comment:
I'm inclined to rename this page to "Syncing and editing course content"?

Reply:
Sounds good to me!

Reply:
Done in 024e52f!